tests/benchmarks/static_message_source.py

```diff
@@ -46,23 +48,11 @@ def supports_cpp_node(self):
     def input_count(self) -> int:
         return len(self._df)
 
-    def _build_source(self, builder: mrc.Builder) -> StreamPair:
-        out_stream = builder.make_source(self.unique_name, self._generate_frames())
-        return out_stream, MessageMeta
-
-    def _post_build_single(self, builder: mrc.Builder, out_pair: StreamPair) -> StreamPair:
-
-        out_stream = out_pair[0]
-        out_type = out_pair[1]
-
-        # Convert our list of dataframes into the desired type. Flatten if necessary
-        if (typing_utils.issubtype(out_type, typing.List)):
-            flattened = builder.make_node(self.unique_name + "-post", ops.flatten())
-            builder.make_edge(out_stream, flattened)
-            out_stream = flattened
-            out_type = typing.get_args(out_type)[0]
```

Comment:
Sure, but it's not a breaking change. The output type of this stage is hard-coded to `MessageMeta` such that the `if (typing_utils.issubtype(out_type, typing.List)):` branch is never executed. 

Since this isn't user-facing production code, I didn't think it needed to be called out.